compute/mailjet/README.md

```diff
@@ -0,0 +1,44 @@
+# Java Mailjet Email Sample for Google Compute Engine
+
+This sample demonstrates how to use [Mailjet](https://www.mailjet.com/) on
+[Google Compute Engine](https://cloud.google.com/compute/)
+
+See the [sample application documentaion][sample-docs] for more detailed
+instructions.
+
+For more information about Mailjet, see their
+[documentation](http://dev.mailjet.com/email-api/v3/apikey/).
+
+[sample-docs]: https://cloud.google.com/compute/docs/tutorials/sending-mail/
+
+## Running on Compute Engine
+
+To run the sample, you will need to do the following:
+
+1. [Create a Mailjet Account](https://app.mailjet.com/signup).
+1. Create a compute instance on the Google Cloud Platform Developer's Console
+1. SSH into the instance you created
+1. Update packages and install required packages
+
+    `sudo apt-get update && sudo apt-get install git-core openjdk-8-jdk maven`
```

Comment:
Should we give them a script that does this?  So they can create their instance and set it up?